code/game/objects/items/pneumaticCannon.dm

```diff
@@ -333,7 +333,8 @@
 	. = ..()
 	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
 	STR.max_items = 30
-	STR.max_combined_w_class = 120
+	STR.max_combined_w_class = 600
```

Comment:
```suggestion
	STR.max_combined_w_class = 300
```

The actual max value, but this isn't super important tbh

Reply:
Needs to fit 30 magspears so it should be 600 since they're bulky

Reply:
Made it not hardcoded